content/docs/get-started/aws/begin.md

```diff
@@ -50,17 +52,37 @@ Next, you'll install the required language runtime, if you haven't already.
 {{< install-dotnet >}}
 {{% /choosable %}}
 
-Finally, you'll configure AWS.
+Finally, you'll configure Pulumi with AWS.
 
-### Configure AWS
+### Configure Pulumi to access your AWS account
 
-<a href="{{< relref "/docs/intro/cloud-providers/aws/setup" >}}" target="_blank">Configure AWS</a> so the Pulumi CLI can connect to AWS. If you have previously configured the <a href="https://aws.amazon.com/cli/" target="_blank">AWS CLI</a>, `aws`, Pulumi will respect and use your configuration settings.
+Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user account that has **Programmatic access** with rights to deploy and manage resources handled through Pulumi. In this guide, your IAM user must have the `s3:CreateBucket` IAM policy action granted.
 
-If you have multiple AWS profiles set up, specify a different profile using one of the following ways:
+If you have previously <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html" target="_blank">installed</a> and <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html" target="_blank">configured</a> the AWS CLI, Pulumi will respect and use your configuration settings.
+
+If you do not have the AWS CLI installed or plan on using Pulumi from within a CI/CD pipeline, <a href="https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys" target="_blank">retrieve your access key ID and secret access key</a> and then set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables on your workstation.
+
+#### Linux/macOS
+
+```bash
+$ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+$ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+#### Windows
+
+```bat
+> set AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+> set AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+As an optional step, if you have multiple AWS profiles set up, you can specify a different profile using one of the following methods:
 
 * Set `AWS_PROFILE`as an <a href="{{< relref "/docs/intro/cloud-providers/aws/setup#environment-variables" >}}" target="_blank">environment variable</a>, or
 * After creating your project in the next step, run `pulumi config set aws:profile <profilename>`. See <a href="{{< relref "/docs/intro/cloud-providers/aws#configuration" >}}" target="_blank">AWS Configuration</a> for more configuration options.
 
+For additional information on how to setup Pulumi to work with AWS, see <a href="{{< relref "/docs/intro/cloud-providers/aws/setup" >}}" target="_blank">AWS Setup</a>.
```

Comment:
I think I understand why this targets `_blank`, but in general, we rarely (if ever?) target a new window when linking to another page on our own site, expecting the user to make this choice if they mean to. So would suggest just a regular ol' Markdown link here:

 ```suggestion
For additional information on how to setup Pulumi to work with AWS, see [AWS Setup]({{< relref "/docs/intro/cloud-providers/aws/setup" >}}).
```

Reply:
That's fine with me. I'll make the same changes above then.